src/main/charts/bitbucket/values.yaml

```diff
@@ -451,7 +451,12 @@ bitbucket:
     # -- The GID used by the Bitbucket docker image
     #
     gid: "2003"
-    
+
+  # -- Boolean to define whether to set home directory permissions on startup
+  # of Bitbucket container. Set to 'false' to disable this behaviour.
+  #
+  setPermissions: true
```

Comment:
I was considering whether `setLocalHomePermissions` won't be more explicit but I am not sure if that is not just unnecessary complicating things. It just feels that `setPermissions` can mean anything but you can read the comment to understand it so I guess that should be fine.

Reply:
yea, I agree with you on making it clear on what this variable does. In terms of the naming, I feel there is also some benefits to keep the name consistent (despite the uppercase /camel case thing), especially for developers. So more inclined to the current version.